src/applications/discharge-wizard/containers/FormPage.jsx

```diff
@@ -1,37 +1,37 @@
+// Dependencies
 import { connect } from 'react-redux';
-import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import React from 'react';
 
+// Relative Imports
 import { updateField } from '../actions';
 import FormQuestions from '../components/FormQuestions';
 
-export class FormPage extends Component {
-  componentDidMount() {
```

Comment:
Is this componentDidMount logic not needed anymore?

Reply:
Yes, same as the comment below!